### PR TITLE
docs: multi-party recovery

### PR DESCRIPTION
**Pull request by thomasten**
add documentation for multi-party recovery to the public docs

### CODE REVIEW

File: docs/docs/workflows/recover-coordinator.md [hunk 1]
```diff
@@ -1,53 +1,49 @@
 # Recovering the Coordinator
 
-As described in the [recovery chapter](../features/recovery.md), different situations can require the *recovery* of the Coordinator.
-If the Coordinator finds a sealed state during its startup which it's unable to unseal using the host-specific SGX sealing key, it will wait for further instructions.
+Different situations require the [recovery](../features/recovery.md) of the Coordinator.
+If the Coordinator fails to unseal its state, it will enter recovery mode.
 
-:::note
+You need the corresponding private key to the [`RecoveryKeys` defined in the manifest](define-manifest.md#recoverykeys), and the [recovery secret returned to you during the initial upload of the manifest](set-manifest.md).
```
Comment (m1ghtym0) [inline docs/docs/workflows/recover-coordinator.md:6]:
```suggestion
You need the corresponding private key to the [`RecoveryKeys` defined in the manifest](define-manifest.md#recoverykeys) and the [recovery secret returned to you during the initial upload of the manifest](set-manifest.md).
```

File: docs/docs/workflows/recover-coordinator.md [hunk 2]
```diff
@@ -1,53 +1,49 @@
...
-:::
-
-You have two options:
-
-1. Recover the sealed state by uploading the recovery secret, which was encrypted for the `RecoveryKeys` defined in the manifest.
+:::info
 
-    The recovery secret can be uploaded through the `/recover` client API endpoint. To do so, you need to:
+If you don't want or can't recover the old state, you can also dismiss it by [uploading a new manifest](set-manifest.md).
+The old state will be overwritten on disk and the `/recover` endpoint won't be available anymore.
```
Comment (m1ghtym0) [inline docs/docs/workflows/recover-coordinator.md:11]:
```suggestion
If you don't want or can't recover the old state, you can also dismiss it by [uploading a new manifest](set-manifest.md).
The old state will be overwritten on disk, and the `/recover` endpoint won't be available anymore.
```

File: docs/docs/workflows/recover-coordinator.md [hunk 3]
```diff
@@ -1,53 +1,49 @@
...
+On success, the Coordinator applies the sealed state again. If the Coordinator can't restore the state with the uploaded key, an error will be returned in the logs and the `/recover` endpoint will stay open for further interaction.
 
-:::info
+## Multi-party recovery
 
-If a new manifest is uploaded, the old state will be overwritten on disk and the `/recover` endpoint won't be available anymore.
+<enterpriseBanner/>
 
-:::
+If you've [configured your MarbleRun deployment for multi-party recovery](define-manifest.md#multi-party-recovery), send each party the corresponding [recovery secret](set-manifest.md). Ask them to perform the steps above. Once all parties have uploaded their secrets, the Coordinator recovers the sealed state and continues operation.
```
Comment (m1ghtym0) [inline docs/docs/workflows/recover-coordinator.md:49]:
```suggestion
If you've [configured your MarbleRun deployment for multi-party recovery](define-manifest.md#multi-party-recovery), send each party the corresponding [recovery secret](set-manifest.md). Ask them to perform the steps above. Once all parties have uploaded their secrets, the Coordinator recovers the sealed state and continues its operations.
```

**Review by m1ghtym0 — CHANGES_REQUESTED** [review]
Awesome, thanks!
Just some minor suggestions. Feel free to ignore them if they don't sound right to you.

File: docs/docs/features/recovery.md [hunk 4]
```diff
@@ -16,8 +20,16 @@ The owner of the Recovery Key can access the raw state of the Coordinator.
...
+-->
 
-:::
+## Multi-party recovery
+
+<enterpriseBanner/>
+
+Depending on the use case, it may not be acceptable that the owner has full control over the cluster.
+MarbleRun supports splitting the Recovery Key between parties.
+Only if all parties agree can they recover a cluster or access the raw state.
```
[the remark this answers is not in the document]
Comment (thomasten) [reply docs/docs/features/recovery.md:35]: I decided to keep this based on https://www.grammaring.com/only-after-only-if-only-in-this-way-etc-not-until

**Pull request merged**